src/fondant/pipeline/pipeline.py

```diff
@@ -305,6 +300,10 @@ def _get_registry_path(name: str) -> Path:
             raise ValueError(msg)
         return component_dir
 
+    @property
+    def component_id(self) -> str:
```

Comment:
Is there a reason to call this `id` here instead of `name`?

Reply:
I'll update it